ros/lib/rbac_interface.py

```diff
@@ -0,0 +1,138 @@
+from urllib.parse import urljoin
+from http import HTTPStatus
+from flask_restful import abort
+from .config import RBAC_SVC_URL, PATH_PREFIX
+import requests
+
+
+RBAC_SVC_ENDPOINT = "/api/rbac/v1/access/?application=%s"
+AUTH_HEADER_NAME = "X-RH-IDENTITY"
+VALID_HTTP_VERBS = ["get", "options", "head", "post", "put", "patch", "delete"]
+
+
+def fetch_url(url, auth_header, logger, method="get"):
+    """
+    Helper to make a single request *** Add to this comment once functionality done.
```

Comment:
looks like a TODO item snuck in here. IMO either update the method doc block or just remove the "add to this comment" note if no additional info is needed.

Reply:
done.